### PR TITLE
Contact Form: allow pre-filling form fields with URL parameters

### PR DESCRIPTION
**Pull request by osiux**
POST takes precedence over GET variables, and both methods over user data in the case the user is logged in. For #2554

### CODE REVIEW

**Comment by jeherve** [comment]
Fixes #2554

File: modules/contact-form/grunion-contact-form.php [hunk 1]
```diff
@@ -1769,6 +1769,8 @@ function render() {
 
 		if ( isset( $_POST[$field_id] ) ) {
 			$this->value = stripslashes( (string) $_POST[$field_id] );
+		} elseif ( isset( $_GET[$field_id] ) ) {
```
Comment (dereksmart) [inline modules/contact-form/grunion-contact-form.php]: can you please add spaces in bracket like `$_GET[ $field_id ]`?

**Comment by osiux** [comment]
@dereksmart updated! Also added spaces on the $_POST var above my changes.

**Comment by dereksmart** [comment]
:+1: nice -- looks good

**Pull request merged**